### ISSUE TITLE
Circleci cache detects only setup.py changes

### ISSUE THREAD

**Issue by pblayo** [openfisca/country-template#60]
The problem, after merging of #59 :

```sh
openfisca-core$ make test
rm -rf build dist
find . -name '*.pyc' -exec rm \{\} \;
python -m compileall -q .
flake8 `git ls-files | grep "\.py$"`
pytest
====================================================================== test session starts =======================================================================
platform linux -- Python 3.6.7, pytest-3.9.3, py-1.7.0, pluggy-0.8.0
rootdir: /home/pblayo/openfisca-core, inifile: setup.cfg
collecting 0 items / 1 errors                                                                                                                                    
============================================================================= ERRORS =============================================================================
____________________________________________________________ ERROR collecting tests/core/test_axes.py ____________________________________________________________
tests/core/test_axes.py:5: in <module>
    from openfisca_country_template import CountryTaxBenefitSystem
../.pyenv/versions/3.6.7/envs/openfisca-core-3.6.7/lib/python3.6/site-packages/openfisca_country_template/__init__.py:8: in <module>
    from openfisca_country_template.situation_examples import couple
../.pyenv/versions/3.6.7/envs/openfisca-core-3.6.7/lib/python3.6/site-packages/openfisca_country_template/situation_examples/__init__.py:16: in <module>
    single = parse('single.json')
../.pyenv/versions/3.6.7/envs/openfisca-core-3.6.7/lib/python3.6/site-packages/openfisca_country_template/situation_examples/__init__.py:12: in parse
    with open(file_path, 'r') as file:
E   FileNotFoundError: [Errno 2] No such file or directory: '/home/pblayo/.pyenv/versions/3.6.7/envs/openfisca-core-3.6.7/lib/python3.6/site-packages/openfisca_country_template/situation_examples/single.json'
============================================================== 1 warnings, 1 error in 3.51 seconds ===============================================================
make: *** [test] Error 1
```
in .circle/config.yml : 
```
      - save_cache:
          key: v1-py2-{{ checksum "setup.py" }}
          paths:
            - /tmp/venv/country_template
            - dist
```

**Comment by maukoquiroga** [comment]
Hi @pblayo, this problem _may_ propagate also to `extension-template`, and even `core` or `France`.

**Comment by maukoquiroga** [comment]
Fixed by #62
